### PR TITLE
Clarify the mapping of the pseudo intrinsics

### PR DESCRIPTION
**Pull request by rofirrim**
Also add an example to vcreate to avoid having to complicate the text.

See #321

### CODE REVIEW

File: doc/rvv-intrinsic-spec.adoc [hunk 1]
```diff
@@ -560,66 +560,85 @@ Availability of the tuple types follows the availability of their corresponding
...
+The `vsetvl` intrinsics return the number of elements processed in a stripmining loop when  provided with the element width and LMUL in the intrinsic suffix. This pseudo intrinsic is typically mapped to `vsetvli` or `vsetivli` instructions.
 
-NOTE: The implementation must respect the ratio between SEW and LMUL given to the intrinsic. On the other hand, as mentioned in <<control-of-vl>>, the `vsetvl` intrinsics do not necessarily map to the emission a `vsetvl` instruction of that exact SEW and LMUL provided. The actual value written to the `vl` control register is an implementation defined behavior and typically not known until runtime.
+NOTE: The implementation must respect the ratio between SEW and LMUL given to the intrinsic. On the other hand, as mentioned in <<control-of-vl>>, the `vsetvl` intrinsics do not necessarily map to the emission a `vsetvli` or `vsetivli` instruction of that exact SEW and LMUL provided. The actual value written to the `vl` control register is an implementation defined behavior and typically not known until runtime.
```
Comment (4vtomat) [inline doc/rvv-intrinsic-spec.adoc:570]: Are we missing `vsetvl` instruction?

File: doc/rvv-intrinsic-spec.adoc [hunk 2]
```diff
@@ -560,66 +560,85 @@ Availability of the tuple types follows the availability of their corresponding
...
 === `vlmul_trunc`
 
-The `vlmul_trunc` intrinsics are syntactic sugars that have the same semantic as `vget` with `idx=0`.
+The `vlmul_trunc` intrinsics are syntactic sugar for RVV vector types other than tuples and have the same semantic as `vget` with the `index` operand having the value `0`.
 
 [[pseudo-vlmul_ext]]
 === `vlmul_ext`
 
-The `vlmul_ext` intrinsics are syntactic sugars that have the same semantic as `vset` with `idx=0`.
+The `vlmul_ext` intrinsics are syntactic sugar for RVV vector types other than tuples and have the same semantic as `vset` with `index` operand having the value `0`.
```
Comment (4vtomat) [inline doc/rvv-intrinsic-spec.adoc:613]: Maybe `index` -> `the index` same as the description above?

File: doc/rvv-intrinsic-spec.adoc [hunk 3]
```diff
@@ -560,66 +560,85 @@ Availability of the tuple types follows the availability of their corresponding
...
+dest = __riscv_vset_v_f32m2_f32m4(__riscv_vundefined_f32m4(), 0, v0);
+dest = __riscv_vset_v_f32m2_f32m4(dest, 1, v1);
+----
+====
 
 [[pseudo-vlenb]]
 === `vlenb`
 
-The `vlenb` intrinsic returns what is held inside the read-only CSR `vlenb` ^29^, which is the vector register length in bytes.
+The `vlenb` intrinsic returns what is held inside the read-only CSR `vlenb` ^29^, which is the vector register length in bytes. This pseudo intrinsic is mapped to a `csrr` instruction that reads from the `vlenb` operand.
```
Comment (4vtomat) [inline doc/rvv-intrinsic-spec.adoc:641]: `operand` -> `CSR`?

File: doc/rvv-intrinsic-spec.adoc (hunk 1 above)
Comment (nick-knight) [reply doc/rvv-intrinsic-spec.adoc:570]: The omission is intentional: `vtype` is statically encoded in the API.

**Review by kito-cheng — APPROVED** [review]
LGTM :)

**Pull request merged**